doc/Accesskeys.md

```diff
@@ -1,81 +1,88 @@
-Accesskeys in Friendica
+Accesskeys reference
 =======================
 
 * [Home](help)
 
-For an overview of the modifier key of the different browsers we suggest this [Wikipedia](https://en.wikipedia.org/wiki/Access_key) article.
+Access keys are keyboard shortcuts that allow you to easily navigate the user interface.
+Access keys are currently not available with the Frio theme.
+
+The specific key combinations depend on how your browser's the modifier key setting.
+For an overview of modifier keys in different browsers, have a lookat [Wikipedia](https://en.wikipedia.org/wiki/Access_key) article.
+For example, for moving to profile page in Firefox, press these four keys simultaneously.
+
+[Shift] [Alt] [:] [p]
```

Comment:
What is the `:` for?. I can just use Shift+Alt+p in Firefox running on Debian here.

Reply:
Ha, yes! 

I'll fix it right now.